requirements.txt

```diff
@@ -23,6 +23,7 @@ tweepy
 click
 spacy>=3.0.0,<4.0.0
 en-core-web-sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.5.0/en_core_web_sm-3.5.0-py3-none-any.whl
+pymilvus
```

Comment:
I'm not sure it is a good idea to include this if the Milvus version that the user is running is unknown, since they should be compatible.. What do you think?